src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
```

Comment:
Oh... Ya I didn't like using that particular scope. But if you look at why
it is there either I am missing something important or it is just not
possible another way.

On Sep 7, 2016 3:08 PM, "Maurizio Casimirri" <notifications@github.com>
wrote:

> In src/js/components/navbars.js
> <https://github.com/mcasimir/mobile-angular-ui/pull/378#discussion_r77913520>
> :
>
> > @@ -85,48 +86,127 @@
> >
> >    var module = angular.module('mobile-angular-ui.components.navbars', []);
> >
> > -  /**
> > -   * @directive navbarAbsoluteTop
> > -   * @restrict C
> > -   * @description
> > -   *
> > -   * Setup absolute positioned top navbar.
> > -   *
> > -   * ``` html
> > -   *  <div class="navbar navbar-app navbar-absolute-top">
> > -   *    <!-- ... -->
> > -   *  </div>
> > -   * ```
> > -   */
> > +  var global;
>
> Its a code review, the comment is bound to a line on the diff. So each
> comment is specific to one line.
>
> I was referring to var global specifically.
>
> A name like var globalNavbarControllerSingleton would have been better.
> Long names are not bad, although the usually feel strange.
>
> Abbreviations and generic names can not be that obvious. Immagine i didn't
> even know what that line was related to. That may happen to someone editing
> your code after you.. in order to understand what is var global there for
> he/she should have read and understood all the rest of the code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mcasimir/mobile-angular-ui/pull/378/files/ccdae045247c4f5e44c42d1cbd5319eb3fcb252f#r77913520>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADDzgwhQ2TwLIXy5xd3c_xWKlkYFOzR8ks5qnzXMgaJpZM4Jx8nS>
> .
>